src/game_api/level_api.hpp

```diff
@@ -170,8 +170,8 @@ struct DoorCoords
 class ThemeInfo
 {
   public:
-    bool unknown1; // gets set to false for the jungle and temple theme in the cosmic ocean
-    bool unknown2;
+    bool allow_beehive;
+    bool allow_leprechaun;
```

Comment:
Aren't leprechaun's unique to the "voice echoes" levels? you sure this is specific to the leprechaun alone?